db/migrate/20160530184400_add_repo_id_indexes.rb

```diff
@@ -0,0 +1,10 @@
+class AddRepoIdIndexes < ActiveRecord::Migration
+  def up
+    add_index :katello_repository_docker_manifests, :repository_id
+    add_index :katello_repository_errata, :repository_id
+    add_index :katello_repository_ostree_branches, :repository_id
+    add_index :katello_repository_package_groups, :repository_id
+    add_index :katello_repository_puppet_modules, :repository_id
+    add_index :katello_repository_rpms, :repository_id
+  end
+end
```

Comment:
shouldn't we also include a `#down` method? i realize that people will rarely if ever migrate backwards, but i feel it's good practice to make sure migrations are happy in both directions.